misc_scripts/add_allele_registry_link.rb

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env ruby
+
+require "uri"
+require 'digest/sha1'
+
+
+# create URL with authentication
+# params = vector of param=value
+def createUrl2(hostPath, params, login, password)
+	# encode parameters
+	params2 = []
+	params.each { |p|
+		i = p.index('=')
+		next if i.nil? or (i+1 == p.size)
+		p = p[0..i] + URI::escape(p[i+1..-1])
+		params2 << p
+	}
+	# build url
+	urlSuffix = "?#{params2.join('&')}"
+    hostPathCoded = hostPath.split('/')
+    hostPathCoded[0].sub!(/:\d+$/,'')   # remove port number
+    hostPathCoded = hostPathCoded.join('/')
+    urlCoded = "http://" + hostPathCoded + urlSuffix
+    url      = "http://" + hostPath      + urlSuffix
+    identity = Digest::SHA1.hexdigest("#{login}#{password}")
+    gbTime = Time.now.to_i.to_s
+    token = Digest::SHA1.hexdigest("#{urlCoded}#{identity}#{gbTime}")
+    request = "#{url}&gbLogin=#{URI::escape(login)}&gbTime=#{gbTime}&gbToken=#{URI::escape(token)}"
+    return request
+end
+
+def createUrl(hostPath, params)
```

Comment:
Again a minor nitpick, but generally in ruby method names are `snake_case` rather than `camelCase`.